docs/deploy.rst

```diff
@@ -0,0 +1,228 @@
+
+.. _deploy:
+
+======
+Deploy
+======
+
+To deploy a CrateDB cluster, you first need an organization and project, and
+this page takes you through each of the steps. You will:
+
+- Create an organization
+- Set a default region in Croud (determines where your data is stored)
+- Create a project
+- Check for available CrateDB products in your default region
+- Deploy a CrateDB cluster
+
+.. rubric:: Table of contents
+
+.. NOTE::
+
+  This document assumes that you're `logged in`_ with your CrateDB Cloud user
```

Comment:
"This document assumes that you have created a CreatDB Cloud account and that you are logged in with the Croud CLI tool." (links as appropriate)